Homework_3/task3/templates/login.html

```diff
@@ -0,0 +1,37 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    {% block title %}
+    {{ super() }} - {{ title }}
+    {% endblock %}
+
+    <h1>Login</h1>
+    <div class="container">
+    <form action="{{ url_for('login') }}" method="POST">
```

Comment:
Атрибут `action` можно не указывать, так как шаблон контролируется функцией преставления
```suggestion
    <form method="POST">
```